kubelet/datadog_checks/kubelet/prometheus.py

```diff
@@ -125,6 +125,10 @@ def _is_pod_metric(labels):
         if 'container_name' in labels:
             if labels['container_name'] == 'POD':
                 return True
+        # Some runtimes does not report container_name="POD"
+        if 'pod_name' in labels:
```

Comment:
For the metric to be a pod-level one, don't you also need container_name=""? (because all container metrics also have a pod_name label)

Reply:
Good catch, just fixed it